src/restore/dataset/dataset_restore_service.py

```diff
@@ -24,7 +24,10 @@ def restore(self, restoration_job_id, project_id, dataset_id,
             max_partition_days)
 
         try:
-            RestorationJob.create(restoration_job_id)
+            restoration_job_key = RestorationJob.create(
+                restoration_job_id,
```

Comment:
Pass name parameter in all of those or none (e.g. create_disposition=)

Reply:
I will do in next pull request when I will edit whole dataset restore service